### ISSUE TITLE
TOOD proxy automatically reconnects to Email server in case of a disconnect

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#61]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by M-HRL** [comment]
### The CI Output:

![Image](https://github.com/amosproj/amos2023ws01-ticket-chat-ai/assets/60758809/5fbe9ed1-dfb8-43ad-8aed-5a209d6a25af)
